src/main/java/fr/insalyon/citi/golo/runtime/MethodInvocationSupport.java

```diff
@@ -159,6 +160,26 @@ public static Object fallback(InlineCache inlineCache, Object[] args) throws Thr
     Class<?> receiverClass = args[0].getClass();
     MethodHandle target = lookupTarget(receiverClass, inlineCache, args);
 
+    if (target == null) {
```

Comment:
My constraints:
* the `fallback` resolution is made only when the first resolution failed.
  * Easy to handle in a strategy
* if the `fallback` has to be invoked then you have to create a specific callsite 
  * `findTarget` return `MethodHandle` this is the tricky part
* If the `fallback` resolution failed then the displayed message should only mention the initial method invoked (not the fallback one)
  * Easy